policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -38,250 +54,412 @@ policyengine_to_taxsim:
+    group_column: 2
   v29:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "29. FICA \n  Taxpayer share of FICA"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v30:
     variable: household_net_income
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "30. Household Income"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v31:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "31. Imputed Rent"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v32:
     variable: state_agi
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "32. AGI"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v33:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "33. Exemptions"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v34:
     variable: state_standard_deduction
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "34. Standard Deduction"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v35:
     variable: state_itemized_deductions
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "35. Itemized Deductions"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v36:
     variable: state_taxable_income
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "36. Taxable Income \n  Tax before credits"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v37:
     variable: state_property_tax_credit
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "37. Property Tax Credit"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v38:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "38. Child Care Credit"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v39:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "39. EIC \n Energy|Fuel Credit \n Child Tax Credit"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v40:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "40. Total Credits"
```

Comment:
this should be a sum of
'state_non_refundable_credit'
and 
'state_refundable_credits'